source/client/player/input/ControllerTurnInput.cpp

```diff
@@ -22,6 +22,9 @@ ControllerTurnInput::ControllerTurnInput()
 
 ITurnInput::Delta ControllerTurnInput::getTurnDelta()
 {
+#ifdef __ANDROID__
+	return Delta(Controller::getX(m_stickNo) * 50.f, Controller::getY(m_stickNo) * 60.f);
```

Comment:
Why 50 and 60? Magic values much?

Note: this code was written for the xperia play, so you will want to modify it, but I'm just checking that it's modified correctly

Reply:
Yeah.... its just some values to increase turn sensitivity it should probably be implement in Options